util/dvsim/doc/testplanner.md

```diff
@@ -40,14 +40,36 @@ The following attributes are used to define each testpoint, at minimum:
 
     Full [Markdown]({{< relref "doc/rm/markdown_usage_style" >}}) syntax is supported when writing the description.
 
-* **tests: list of written test(s) for a testpoint**
+* **tests: list of written test(s) for this testpoint**
 
     The testplan is written in the initial work stage of the verification [life-cycle]({{< relref "doc/project/development_stages#hardware-verification-stages" >}}).
     Later, when the DV engineer writes the tests, they may not map one-to-one to a testpoint - it may be possible that a written test satisfactorilly addresses multiple testpoints; OR it may also be possible that a testpoint needs to be split into multiple smaller tests.
     To cater to these needs, we provide the ability to set a list of written tests for each testpoint.
     It is used to not only indicate the current progress so far into each milestone, but also map the simulation results to the testpoints to generate the final report table.
     This list is initially empty - it is gradually updated as tests are written.
 
+* **tags: list of tags relevant for this testpoint**
+
+    Tags represent the need to run the same testpoint under specific conditions, in additional platforms or with specific set of build / runtime options.
```

Comment:
Maybe it's worth adding (or referencing the description of)  the default test environment, e.g. DV sim, with test ROM as first boot stage, and test as second boot stage?